pkg/ingester/ingester.go

```diff
@@ -502,6 +556,9 @@ func (i *Ingester) updateLoop(ctx context.Context) error {
 		case <-usageStatsUpdateTicker.C:
 			i.updateUsageStats()
 
+		case <-resourceUtilizationUpdateTicker.C:
```

Comment:
We expect the resources utilization to be updated at an high frequency. Bundling it with all other updates may lead to the resources utilization update to be delayed if other updates are slow (e.g. think what happens if it takes 10s to run `updateActiveSeries()`). Is this a real issue? I don't know. Looking at what we do in this `for` loop, there shouldn't be anything taking longer than a few seconds in the worst case, so it shouldn't be an issue right now.

However, a safer approach may be moving the "limiting" logic to a dedicated `services.Service` (so running at its own schedule) which then we add to `Ingester.subservices`.

Reply:
Agreed, @pracucci. Since it's a time sensitive procedure, it's safer to have in its own sub-service. Will take care of it.

Reply:
Done.